src/main/java/org/fundacionjala/core/ui/CommonActions.java

```diff
@@ -21,6 +21,7 @@ private CommonActions() {
      * wait and click action.
      * @param element webelement.
      */
+    //TODO rename methods
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)